app/components/FileLoader.js

```diff
@@ -413,6 +415,20 @@ export default class FileLoader extends Component {
       </Table>
     );
   }
+  
+  deleteSelections = () => {
```

Comment:
Actions like this should be done in redux. With the current logic, if anything else ever updates the redux stores, I'm pretty sure all those deleted files will come back. Currently I'm not too sure if there's an action that will actually do that but one could be added at any time.

Look at how redux actions are fired in other components and put this handler as an action.

Reply:
OK, I'll learn redux lol